Riot/Views/RoomList/RoomTableViewCell.xib

```diff
@@ -17,7 +17,7 @@
             <rect key="frame" x="0.0" y="0.0" width="600" height="74"/>
             <autoresizingMask key="autoresizingMask"/>
             <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="L2L-l5-wPx" id="aXz-IR-jj5">
-                <rect key="frame" x="0.0" y="0.0" width="600" height="74"/>
+                <rect key="frame" x="0.0" y="0.0" width="600" height="73.5"/>
```

Comment:
I think we should keep 74 for the height. 
No change seems required for this xib file